src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+
+As mentioned, the data availability problem concerns the ability to verify availability of the transaction data for a newly proposed block. Solutions to this problem employ some mechanism for guaranteeing data availability. 
+
+### Data availability committees {#data-availability-committees}
+
+Pure validiums store transaction data off-chain with a block producer, making them centralized to an extent. This reduces decentralization and security, since the block producer can publish invalid transactions and conceal the rollup's true state by hiding transaction data. 
+
+Some validiums mitigate this problem by forcing block producers to store transaction data with trusted parties that form the Data Availability Committee (DAC). Members of the DAC are expected to securely store the data off-chain and publish it on-chain in event of a dispute. They also publish attestations on-chain to prove that the said data is indeed available. 
+
+[More on data availability committees.](https://medium.com/starkware/data-availability-e5564c416424)  
+
+### Proof-of-stake data availability committees {#proof-of-stake-data-availability-committees}
+
+While data availability committees are better than the status quo in a validium, trust assumptions still persist. What if the DAC colludes with the block producer to withhold transaction data? DACs are often small in size, increasing the risk of collusion and the possibility of an external actor compromising the group. 
+
+Some validiums replace DACs with a proof-of-stake (PoS) validator system. Here, anyone can become a validator and store data off-chain. However, they must provide a “bond”, which is deposited in a smart contract. In the event of malicious behavior, such as the validator withholding data, the bond can be slashed. 
+
+Proof-of-stake data availability committees are considerably more secure than regular DACs. Not only are they permissionless and trustless, but they also have well-designed incentives to encourage honest behavior. 
+
+[More on proof-of-stake data availability committees.](https://blog.matter-labs.io/zkporter-a-breakthrough-in-l2-scaling-ed5e48842fbf) 
+
+### Data availability sampling {#data-availability-sampling}
+
+Data availability sampling is a cryptographic mechanism for guaranteeing data availability. Data availability sampling allows blockchain nodes to verify that data for a proposed block is available without having to download the entire block. 
+
+In a DAS system, a node samples small, random chunks of a block over multiple rounds to verify data availability. With many nodes sampling different parts of a block simultaneously, its  availability can be verified with high statistical certainty. 
+
+When applied to blockchains, like Ethereum, data availability sampling ensures light clients also participate in guaranteeing the chain's security and functionality. Light clients can run without expensive hardware, making it easier for anyone to validate on the Ethereum network. 
```

Comment:
Should we indicate that this is the solution we're leaning toward?

Reply:
Based on @jmcook1186's suggestion, I added a new section to explain how Ethereum is approaching data availability (i.e., by using DAS).